controls/srg_ctr/SRG-APP-000516-CTR-001325.yml

```diff
@@ -17,7 +17,6 @@ controls:
   - api_server_admission_control_plugin_service_account
   - api_server_anonymous_auth
   - api_server_api_priority_flowschema_catch_all
-  - api_server_api_priority_gate_enabled
```

Comment:
From a STIG perspective, do we want to remove this right away or wait until a quarterly update?

Reply:
are there any concerns here if we remove it now?

Reply:
Typically - DISA gives us the things we need to change, so we maintain alignment with what they publish.

Reply:
that makes sense, just revert the change there

Reply:
I agree with keeping these rules in the STIG profiles until the next update.

@Vincent056 @rhmdnd Note that the STIG profiles for OCP are not using the `srg_ctr` controls.
They are now leveraging `stig_ocp4` control file.

And strictly speaking `SRG-APP-000516-CTR-001325` is not part of the OCP4 STIG, so these rules were added by us.

Reply:
On a second thought, these rules resulting in NOT-APPLICABLE in the supported OCP versions and have not effect at all.
@Vincent056 I think we can remove these rules from the STIG profile as well.